apps/api/src/domain/models/shared/common-command-errors/AggregateIdNotAvailableError.ts

```diff
@@ -8,6 +8,6 @@ import { AggregateId } from '../../../types/AggregateId';
  */
 export default class AggregateIdNotAvailableError extends InternalError {
     constructor(invalidId: AggregateId) {
-        super(`The ID: ${invalidId} is not available for use`);
+        super(`The ID: ${invalidId} is not available for use by an aggregate root`);
```

Comment:
I'm not sure why we have a separate error for `AggregateId` not available, but let's not reword this message. Note that the ID is all that will be needed to troubleshoot this upstream anyways.

Reply:
![image](https://github.com/user-attachments/assets/dd1eae4b-8786-4f45-a2d4-61132c4179b1)
It was unused. I have removed this.